p2p/server.go

```diff
@@ -836,6 +836,9 @@ running:
 
 func (srv *Server) postHandshakeChecks(peers map[enode.ID]*Peer, inboundCount int, c *conn) error {
 	switch {
+	case discfilter.BannedDynamic(c.node.ID()):
```

Comment:
Wouldn't this had already run with the previous filter you did above?

Reply:
`addPeerChecks()` will repeat the `postHandshakeChecks()` because there can be changes in peerset